contracts/yieldStrategy/gmx/GlpStakingManager.sol

```diff
@@ -153,12 +153,16 @@ contract GlpStakingManager is RageERC4626, OwnableUpgradeable {
     }
```

Comment:
We are depositing using default vault, but checking glpbalance using the list of vaults, which does not contain default vault.